src/views/Upload.vue

```diff
@@ -39,27 +65,131 @@ import UploadTab from '@/components/Upload/UploadTab.vue';
 import UploadFile from '@/components/Upload/UploadFile.vue';
 import UploadText from '@/components/Upload/UploadText.vue';
 
+import axios from 'axios';
+
 export default {
 	components: {
 		Navigation,
 		UploadTab,
 		UploadFile,
-		UploadText,
+		UploadText,	
 	},
 	data: function () {
 		return {
 			selectedTab: '',
 			tabs: ['Nahrať súbory', 'Vložiť text'],
+			files: [],
+			text: '',
+			uploadPercentage: 0,
+			submitted: false,	
+			alert: {
+				message: 'Nezadali ste súbor ani text, ktorý chcete skontrolovať !',
+				type: 'error',
+				duration: 4000,
+			}
 		};
 	},
+	methods: {
+		submitText(){
+			//console.log('text');
+			let formData = new FormData();
+
+			formData.append('file', this.text);
+			formData.append('title','text...');
+
+			if (this.text.length == 0){
+				return this.$store.dispatch('AlertStore/setAlert', this.alert);
+			}
+			axios.post('http://localhost:8000/file/upload/', formData, {
+				headers: {
+					'Content-Type': 'text/plain',
+				},
+				onUploadProgress: function( progressEvent ) {
+					this.submitted = true
+					this.uploadPercentage = parseInt( Math.round( ( progressEvent.loaded * 100 ) / progressEvent.total ) );
+					//console.log(this.uploadPercentage);					
+				}.bind(this)
+			})
+			.then((res) => {
+				console.log('SUCCESS!!' + res);
+			})
+			.then(() =>{
```

Comment:
Why is this in separate `then`s?  Also, you don't return async function from the then above, there's not point on chaining two `then`s. Please remove the console log above and use only one then unless needed.